go.work

```diff
@@ -1,4 +1,4 @@
-go 1.23.0
+go 1.23.4
```

Comment:
is this bump required?  `.0` was specified because it affects downstream

Reply:
Starting in go 1.23.0 the final patch digit needs to be supplied, otherwise `go mod tidy` will result in differences in the go mod which causes the CI to fail.

That said, this version does not need to be updated.  The toolchain will always pull in the latest patch version.

Reply:
```shell
% service % go mod tidy -go=1.23.0
go: github.com/bufbuild/protovalidate-go@v0.9.3 requires go@1.23.4, but 1.23.0 is requested
```
Unfortunately 1.23.4 is required for these dependencies.